### PR TITLE
Linting for tests

### PR DESCRIPTION
**Pull request by captbaritone**

### CODE REVIEW

File: web/static/js/__tests__/SidebarCategory-test.jsx [hunk 1]
```diff
@@ -12,13 +12,13 @@ var TEST_CATEGORY = {
   "tags": [
     {
       "count": 15,
-      "id": "color-scheme",
```
Comment (spicyj) [inline web/static/js/__tests__/SidebarCategory-test.jsx]: Oh, I added these by hand after copy-pasting some JSON! I guess that's not what our lint rules like. :)

**Comment by spicyj** [comment]
:+1:, though I'd personally prefer switching to require trailing commas.

**Comment by captbaritone** [comment]
@spicyj I wouldn't object :smile: I think trailing commas lead to better diffs, but I figured we might as well be consistent for now.

I know some browsers don't like trailing commas. I assume the transpilation that we do in WebPack makes that a non-issue?

**Comment by captbaritone** [comment]
Obviously for tests, the issue of what browsers do/don't like is even more minimal.

**Comment by spicyj** [comment]
I think our current config doesn't strip trailing commas, but IE9 and up don't mind them and I don't think we support IE8.

**Comment by captbaritone** [comment]
Got it. That sounds about right.

**Comment by captbaritone** [comment]
Merging this for now so that our existing rules pass. We can revisit the rule later.

**Pull request merged**